AUDITING.md

```diff
@@ -0,0 +1,75 @@
+# Auditing
+
+This document defines how Platform Chaos audits chaos events. It lists what is tracked and how it is formatted.
+
+## Tracked Data
+
+When a chaos event is terminated through an *API*, *CLI*, or *Extension*, the following properties are recorded and exported through a JSON object.
+  - An `id` unique to this event instantiation (`guid`). 
+  - A `chaos-type` of enumerable type `initiate`
+  - `user` and `system` that terminated the event
+  - `date` of event initiation (including time `UTC`)
+  - targeted `resources`. This is broken down by `subscriptionID` and then by `resourceGroup`
+  - Chaos `event` information (`eventID`, `eventName`, `eventURL`)
+
+Additionally an *Extension* may record additional information regarding the specificities of the event. This information is stored within an `extension` property.
+
+Every chaos event initiation/termination will be logged. The general structure of an event log is defined below:
+
+- _event log_ (Object)
+  - `id` (String) 
+    - GUID unique to this event instantiation
```

Comment:
what is the expectation wrt collisions here?

Reply:
Idea here was that external tools could reference exact chaos events that did _something_. Would also be useful if we build some sort of UI around this to view the audits.

Reply:
thanks - that doesn't answer my question though 🐈 what i mean is, how are we going to generate these, and is the expectation that a given chaotic event should be uniquely addressable by just id, or are there other components from this model that are necessary to uniquely id an event